addons/compat_ws/compat_ws_nouniformrestrictions/config.cpp

```diff
@@ -0,0 +1,20 @@
+#include "script_component.hpp"
+
+class CfgPatches {
+    class ADDON {
+        name = COMPONENT_NAME;
+        units[] = {};
+        weapons[] = {};
+        requiredVersion = REQUIRED_VERSION;
+        requiredAddons[] = {
+            "ace_nouniformrestrictions"
+        };
```

Comment:
```suggestion
        requiredAddons[] = {
            "data_f_lxWS_Loadorder",
            "ace_nouniformrestrictions"
        };
```

Just to be safe?

Reply:
Can subconfig be loaded without the main config being loaded?

Reply:
/shrug